.circleci/config.yml

```diff
@@ -9,7 +9,7 @@ parameters:
   docker_img_version:
     # Docker image version for running tests.
     type: string
-    default: "a0d53ea"
+    default: "03a594c"
```

Comment:
This is a more recent image -- a few dependencies were added, though they don't impact this codebase (only `learned-planners`).